packages/xo-web/src/common/editable/index.js

```diff
@@ -343,70 +334,32 @@ export class Number extends Component {
 
 export class Select extends Editable {
   static propTypes = {
-    options: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
-      .isRequired,
     renderer: PropTypes.func,
+    value: PropTypes.oneOfType([null, PropTypes.object]),
   }
 
-  componentWillReceiveProps (props) {
-    if (
-      props.value !== this.props.value ||
-      props.options !== this.props.options
-    ) {
-      this.setState({
-        valueKey: findKey(props.options, option => option === props.value),
-      })
-    }
-  }
-
-  get value () {
-    return this.props.options[this.state.valueKey]
-  }
-
-  _onChange = event => {
-    this.setState({ valueKey: getEventValue(event) }, this._save)
+  get value() {
+    return this.props.value
   }
 
-  _optionToJsx = (option, key) => {
-    const { renderer } = this.props
-
-    return (
-      <option key={key} value={key}>
-        {renderer ? renderer(option) : option}
-      </option>
-    )
-  }
-
-  _onEditionMount = ref => {
-    // Seems to work in Google Chrome (not in Firefox)
-    ref && ref.dispatchEvent(new window.MouseEvent('mousedown'))
-  }
-
-  _renderDisplay () {
+  _renderDisplay() {
     const { children, renderer, value } = this.props
-
-    return children || <span>{renderer ? renderer(value) : value}</span>
-  }
-
-  _renderEdition () {
-    const { saving, valueKey } = this.state
-    const { options } = this.props
-
     return (
-      <select
-        autoFocus
-        className={classNames('form-control', styles.select)}
-        onBlur={this._closeEdition}
-        onChange={this._onChange}
```

Comment:
You need to keep using the `_save` method so it can have the undo feature.
Also make sure that it behaves exactly as expected. Here, just by testing it, you can notice that the edition doesn't close when you select a new value (which is normal because it's also handled by `_save`.